packages/mathtype-html-integration-devkit/src/parser.js

```diff
@@ -318,10 +318,9 @@ export default class Parser {
    * @returns {Object} JSON object containing showimage response.
    */
   static createShowImageSrc(data, language) {
-    const dataMd5 = [];
+    const dataMd5 = {};
```

Comment:
I'm probably wrong, but this variable shouldn't be defined using 'let' instead of 'const'?

Reply:
True